src/main/clojure/jarl/builtins/opa.clj

```diff
@@ -0,0 +1,11 @@
+(ns jarl.builtins.opa)
+
+(defn builtin-opa-runtime
+  "Implementation of opa.runtime built-in"
+  {:builtin "opa.runtime" :args-types []}
+  [{bctx :builtin-context}]
+  {"commit" ""
+   ; "config" key not present unless running as server - implement later
+   "env" (:env bctx)
+   ; use Jarl version here later
+   "version" "jarl-dev"})
```

Comment:
Could this be pulled from `project.clj` somehow?

Reply:
We could easily inject something via e.g. a system property... but obviously we'd only be interested in compile/build time here, and in that context I have no idea... but let's look into it at some point.